guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -41,12 +41,12 @@ For more information, see https://access.redhat.com/documentation/en-us/red_hat_
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-clean} all
+# dnf clean all
 ----
 +
 . Optional: Verify that the required repositories are enabled:
 +
 [options="nowrap"]
```

Comment:
```suggestion
[options="nowrap" subs="+quotes,attributes"]
```

This should allow asciidoctor to resolve attributes.

If you would like to work on the "package-install" etc attributes, let's have a chat. It's also been on my agenda for a while now.

Reply:
Thanks! Yes, I would absolutely like to work with attributes more, I need to figure out how they are used in the docs set. I'll bring it up at the next docs meeting, but if you prefer to get in touch through other means, feel free to do so.